src/tabs/tabs.scss

```diff
@@ -121,24 +146,16 @@
     &::after {
       content: '';
       position: absolute;
-      left: 0;
-      bottom: 0;
-      width: 100%;
-      height: 0;
       @media (prefers-reduced-motion: no-preference) {
-        transition: height $iui-speed-fast ease-out;
+        transition: width $iui-speed-fast ease-out, height $iui-speed-fast ease-out;
```

Comment:
I think this one is causing strange switch animation.
![vertiacl-tabs-glitch2](https://user-images.githubusercontent.com/36186912/129674019-3e77bf33-2588-496e-88de-56028480fabf.gif)

Reply:
Horizontal Pill tabs also look strange with `iui-not-animated`. Maybe we don't need any transition effect then.

Reply:
I changed the width and height transitions to use `ease` instead of `ease-out` which caused a laggy visual effect.

Reply:
`ease` fixed it in vertical+borderless, but the animation still looks strange in vertical+default and all horizontal ones

Reply:
For me it also still looks strange. Do we really need it? We have normal and nicely looking animation, for non-animated we could just turn of animation at all.

Reply:
We certainly need it for default variant because it doesn't have the JS animated stripe.

Reply:
What if we move this transition rule to active state? Then previous active mark would disappear immediately and new one would have animation.

Reply:
I would be ok with moving the animation only to the active tab.

Reply:
I thought this would work, but it seems to have the same effect as before:
```
.iui-active::after {
    @media (prefers-reduced-motion: no-preference) {
      transition: width $iui-speed-fast ease, height $iui-speed-fast ease;
    }
}
```

Reply:
I see there is several places where `transition` is set. Can you check whether they are needed?

Reply:
Bingo!  You are correct, it's been fixed and I'm pushing the change now.